### ISSUE TITLE
NTR: Environmental Sex Determination

### ISSUE THREAD

**Issue by ValenzuelaLab** [Tree-of-Sex/toso#86]
### In which category is your new term request?

- [X] new_definition
- [ ] existing_definition

### Preferred term label

Environmental Sex Determination - ESD

### Synonym type

None

### Textual definition

The establishment of sex triggered by a non-genetic cue, such as temperature or nutrient availability, experienced within a discrete period after conception. In pure environmental sex determination, no consistent genotypic differences exist between the sexes that would permit the diagnosis of genotypic sex of an individual at fertilization. Environmentally-induced biased sex ratios in species with sex chromosomes due to differential gametogenesis, fertilization, embryonic mortality, or differentiation (i.e. sex reversal) are distinct from environmental sex determination, yet important ecologically and evolutionarily.

### ID of term

http://purl.obolibrary.org/obo/GSSO_000107

### Parent term

Sexual Reproduction, Sex determination

### Children term(s)

Temperature-dependent sex determination; pH dependent sex determination; nutrient dependent sex determination; density dependent sex determination

### ORCID Identifier

0000-0003-1148-631X

### Additional context

Definition in http://purl.obolibrary.org/obo/GSSO_000107 is clarified and expanded. The word 'triggered' was added to "The establishment of sex triggered by a non-genetic cue,..." since ESD is about the initial trigger from the environmental signal that is then transduce into responses of a gene regulatory network. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by CEMcDonoughGoldstein** [comment]
Definition:
- suggest changing "pure" to "complete"
- suggest changing "establishment of sex" to "establishment of sexual development" or "the initiation of sexual development pathways"
- suggest changing "permit the diagnosis of genotypic sex" to "permit the inference of adult sex phenotype based on genotype" (would suggest avoiding using terms such as genotypic sex, hormonal sex, etc. in general as it adds confusion to what sex means)

- would remove "Environmentally-induced biased sex ratios in species with sex chromosomes due to differential gametogenesis, fertilization, embryonic mortality, or differentiation (i.e. sex reversal) are distinct from environmental sex determination, yet important ecologically and evolutionarily."

I think this is important context but distracts from the main definition. Maybe it could be a definition for a different term? E.g., sex ratio bias? And then could be linked with a relational term of distinct from?

**Comment by CEMcDonoughGoldstein** [comment]
Might want to add a sentence about incomplete ESD for balance? 

In complete environmental sex determination, no consistent genotypic differences exist between the sexes that would permit the inference of adult sex phenotype at fertilization. _In incomplete environmental sex determination, genotypic differences may be present but the environment can interact with or reverse genetic sex determination._

**Comment by snuggleduck** [comment]
About this part: "Environmentally-induced biased sex ratios in species with sex chromosomes due to differential gametogenesis, fertilization, embryonic mortality, or differentiation (i.e. sex reversal) are distinct from environmental sex determination, yet important ecologically and evolutionarily." I don't think I agree that sex reversal cannot be a kind of environmental sex determination. The overwriting of genotypic sex determination by environmental stimuli is "triggered by a non-genetic cue, such as temperature or nutrient availability, experienced within a discrete period after conception". There is no gonad differentiation yet when this happens. I agree that in many aspects this is indeed different from "pure ESD", but it is also very different from "pure GSD", so we might want to have an additional term like "environmentally sensitive genotypic sex determination" (E × G) as a more precise definition for "sex reversal", and cite it here.

**Comment by ValenzuelaLab** [comment]
Great comments! Thanks.

I suggest changing the current text to:

The developmental commitment to a sexual fate triggered by a non-genetic cue, such as temperature or nutrient availability, experienced within a discrete period after conception. In fully environmental sex determination, no consistent genotypic differences exist between the sexes that would permit the inference of the genotypic sex of an individual at fertilization. 

Perhaps add this at the end of the definition "Distinct from environmentally-sensitive genotypic sex determination."

And then we could add a new term "Environmentally-sensitive genotypic sex determination": Environmentally-induced biased sex ratios in species with genotypic sex determination due to differential gametogenesis, differential fertilization, differential embryonic mortality, or sex reversal (when information from sex chromosomes or polygenic systems is overriden by an environmental factor). While distinct from fully environmental sex determination, environmentally-sensitive GSD is important ecologically and evolutionarily.

**Comment by snuggleduck** [comment]
I like the previous suggestion (re-iterating: "The developmental commitment to a sexual fate triggered by a non-genetic cue, such as temperature or nutrient availability, experienced within a discrete period after conception. In fully environmental sex determination, no consistent genotypic differences exist between the sexes that would permit the inference of the genotypic sex of an individual at fertilization.")

About the additional issues: I don't think we should lump together sex-determination processes with other processes that procude sex-ratio biases in offspring. When the environment experienced in a critical window overwrites the effect of genotypic sex, we get sex reversal, which is a process that determines the phenoypic sex. The other processes listed here are not sex-determining processes that happen in an individual, but rather they are sex-allocation strategies whereby the parent can manipulate offspring sex ratios (differential gametogenesis, differential fertilization, differential embryonic mortality).

Cibele has just said in her presentation at the workshop that it is not fortunate to have comments in the definition (if I understood her correctly). So instead, we could create definitions for all the other phenomena that are NOT "pure" ESD, e.g.:

Environmental sex reversal: In species with genotypic sex determination, environmental conditions experienced during within a discrete period after conception override genotypic sex to trigger the development of the sexual phenotype that is mismatched with genotypic sex.

Environmentally driven sex allocation: Processes that result in different primary or secondary sex ratios in different environments. Includes sex determination processes in individual offspring (environmental sex determination, environmental sex reversal) and sex-allocation manipulations by parents that result in altered sex ratios in their offspring (e.g. differential gametogenesis, differential fertilization, differential embryonic mortality).

**Comment by mrovatsos** [comment]
i think we should keep the definition simple and straightforward. In children terms, we can give definitions of special cases and subcategories. I would propose:

Sex determination is mediated by environmental factors during a sensitive stage of gonad development, in species without sex-specific sequence differences in their genome.


i would keep GSD with environmental effects as another subcategory, as well as temperature, pH, population density or nutrients,

**Comment by lukkrat** [comment]
> Sex determination is mediated by environmental factors during a sensitive stage of gonad development, in species without sex-specific sequence differences in their genome.

I agree. With sligh polishing: "A sex-determination system in which sex is established by environmental factors during a sensitive stage of gonad development, in species lacking sex-specific sequence differences in their genome."

**Comment by ValenzuelaLab** [comment]
To avoid using the term to be defined as part of the definition, I would suggest another adjustment:

"The establishment of sex by environmental factors during a sensitive stage of gonadal development in species lacking sex-specific genotypic differences"

**Comment by snuggleduck** [comment]
In this context, I would rather call "sex determination" a "process", not a "system". For example, in a mixed sex-determination system, we can have individuals in which the process of sex determination is purely genotypic, while in others it is modified by the environment. Following this line of thought, should we have a definition for sex determination as a process (that happens within individuals or cells) and sex determination as a system (that characterizes a population and all the processes that occur there)?

**Comment by ValenzuelaLab** [comment]
I suggest a further simplification, replacing during a sensitive stage of gonadal development, to during a sensitive stage of development:

"The establishment of sex by environmental factors during a sensitive stage of development in species lacking sex-specific genotypic differences"

**Comment by snuggleduck** [comment]
Nice and to the point. I just wonder: does this definition exclude sex change in sequential hermaphrodites? If we want to exclude that, we could add "gonochoristic": "The establishment of sex by environmental factors during a sensitive stage of development in gonochoristic species lacking sex-specific genotypic differences".
